src/test/regress/sql/multi_colocation_utils.sql

```diff
@@ -57,11 +57,6 @@ CREATE FUNCTION shards_colocated(bigint, bigint)
     AS 'citus'
     LANGUAGE C STRICT;
 
-CREATE FUNCTION get_colocated_table_array(regclass)
```

Comment:
I didn't mean to ask for this specific function.

I think we should have a regression test for repairing co-located shard placements. Do we have any?